### PR TITLE
allow url scheme customized from flutter

### PR DESCRIPTION
**Pull request by dariuspo**

### CODE REVIEW

File: packages/stripe_android/android/src/main/kotlin/com/reactnativestripesdk/StripeSdkModule.kt [hunk 1]
```diff
@@ -219,7 +219,7 @@ class StripeSdkModule(reactContext: ReactApplicationContext, cardFieldManager: S
     val stripeAccountId = getValOr(params, "stripeAccountId", null)
     val urlScheme = getValOr(params, "urlScheme", null)
     val setUrlSchemeOnAndroid = getBooleanOrFalse(params, "setUrlSchemeOnAndroid")
```
Comment (jonasbark) [inline packages/stripe_android/android/src/main/kotlin/com/reactnativestripesdk/StripeSdkModule.kt:221]: I think what you really want is to add setUrlSchemeOnAndroid  in stripe.dart and get rid of the change below

File: packages/stripe/pubspec.yaml [hunk 2]
```diff
@@ -11,9 +11,13 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  stripe_android: ^1.0.1+3
+  stripe_android:
```
Comment (jonasbark) [inline packages/stripe/pubspec.yaml:14]: this should be reverted

**Comment by remonh87** [comment]
@dariuspo  are you still planning to update the pr?  I think if you address @jonasbark  comments we can integrate it.

**Comment by remonh87** [comment]
thanks for your contribution we incorporated the changes in the `sync` branch. It will be part of the new release,  pr can be closed now.

**Pull request closed without merging**